core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -36,6 +36,15 @@
     private LogLevel requestLogLevel = LogLevel.TRACE;
     private LogLevel successfulResponseLogLevel = LogLevel.TRACE;
     private LogLevel failedResponseLogLevel = LogLevel.WARN;
+    private Function<? super RequestLog, LogLevel> requestLogLevelMapper =
+            log -> requestLogLevel();
+    private Function<? super RequestLog, LogLevel> responseLogLevelMapper =
+            log -> log.responseCause() == null ? successfulResponseLogLevel() : failedResponseLogLevel();
```

Comment:
I think we can make theses three methods as private now.
`requestLogLevel()`, `successfulResponseLogLevel()` and `failedResponseLogLevel()`.
But before that let's make them deprecated first.

Reply:
@minwoox Do you have a plan to offer the mapper only in the future? I think these methods can set as a default value of the mapper function.

Reply:
I don't think we are going to remove the three setters and they should not be deprecated.

Reply:
What he's saying about deprecation are the getters.

Reply:
Exactly, sorry for making you confused. 😉

Reply:
Ah. you mean getter() only right?

Reply:
Umm... I agree with @trustin. the deprecation of them is too early decision now.

Reply:
Pushed the commit about deprecation of three `getters`.